site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1530,6 +1558,26 @@ components:
                     format: byte
                     description: The secret key shared between device and Hono.
 
+      AsymmetricKeySecret:
+         additionalProperties: false
+         allOf:
+            - $ref: '#/components/schemas/CommonSecret'
+            - type: object
+              additionalProperties: false
+              required:
+                 - alg
+                 - key
+              properties:
+                 "alg":
+                    type: string
+                    description: The signing algorithm of the provided key (supported algorithms are `RSA` and `EC`).
+                 "key":
+                    type: string
+                    description: |
+                       The Base64 encoded bytes representing the public key / certificate wrapped in a header and footer.
+                       Header and footer (either `-----BEGIN PUBLIC KEY-----` and `-----END PUBLIC KEY-----` for public
+                       keys or `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` for certificates) are mandatory.
```

Comment:
> The Base64 encoded binary DER encoding of the public key.

Reply:
This property does only contain the Base64 encoding of the DER encoded public key. There are no `-----BEGIN PUBLIC KEY-----` headers/footers ...

Reply:
updated